Cue/app/tabs/library/LibraryHome.js

```diff
@@ -66,16 +66,18 @@ class LibraryHome extends React.Component {
 
   constructor(props: Props) {
     super(props)
-
+
+    let refreshing = this.props.decks ? false : true;

```

Comment:
weird, I swore there used to be a componentWillReceiveProps function in library home